bigframes/ml/sql.py

```diff
@@ -170,12 +170,11 @@ def ml_distance(
         col_x: str,
         col_y: str,
         type: Literal["EUCLIDEAN", "MANHATTAN", "COSINE"],
-        source_df: bpd.DataFrame,
+        source_sql: str,
```

Comment:
Is it needed to make this change?

If possible I would rather to let SQL generations stay in sql.py.

Reply:
Generally, want to invert the dependency between domain objects and sql. Will need to further factor things into a common sql generation module anyways (ml and core bigframes have separate sql modules)